packages/assets-controllers/CHANGELOG.md

```diff
@@ -6,6 +6,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.0]
+### Changed
+- Relax dependencies on `@metamask/base-controller`, `@metamask/controller-utils`, `@metamask/network-controller`, and `@metamask/preferences-controller` (use `^` instead of `~`) ([#998](https://github.com/MetaMask/controllers/pull/998))
+- **BREAKING:** Update `onNetworkStateChange`, a constructor option for several controllers, such that it takes an object with a `providerConfig` property instead of `provider` ([#995](https://github.com/MetaMask/controllers/pull/995))
```

Comment:
Nit: usually we place breaking changes first.

Though maybe I should stop making this observation, if we don't lint this? It's not a big deal either way I guess 

Tempting to add a lint for it, but changelog lints are painful now because of the lack of an auto-fix command.

Reply:
Updated in [d73845f](https://github.com/MetaMask/controllers/pull/999/commits/d73845fc49d45ef3be1ff3c1a877d03ad50d4aed).